recipe/build.sh

```diff
@@ -1,18 +1,10 @@
-#!/bin/bash
```

Comment:
Yeah getting rid of `export CONDA_DEFAULT_ENV=''` was a mistake. I see that now. 😞